src/nomos/agent/STRINGS.in

```diff
@@ -6189,15 +6186,15 @@ k
 #####
 %ENTRY% _LT_SEE_COPYING_LICENSE_1
 %KEY% =NULL=
-%STR% "see =FEW= (copyright notices?|license|legal notices?|copying|copyright\.?txt|license\.?txt|readme\.?txt|file) =FEW= (for|with|in|at) =FEW= (details|licens(e|ing)|distribut|information|folder|directory|https?|ftp)"
+%STR% "see =FEW= (copyright notices?|license|legal notices?|copying|copyright\.?txt|license\.?txt|readme\.?txt|file) =FEW= (for|with|in|at) =FEW= (details|licens(e|ing)|distribut|information|foder|directory|https?|ftp)"
 #
 %ENTRY% _LT_SEE_COPYING_LICENSE_2
 %KEY% =NULL=
 %STR% "for =FEW= (details|license|distribut|information|use|usage) =FEW= see =FEW= (copyright notices?|license|legal notices?|copying|copyright\.?txt|license\.?txt) =FEW= (for|with|in|at)"
 #
 %ENTRY% _LT_SEE_URL
 %KEY% =NULL=
-%STR% "see (https?|ftp) =SOME= (for|with|in) =FEW= (details|license|distribut|information|folder|directory)"
+%STR% "see (https?|ftp) =SOME= (for|with|in) =FEW= (details|license|distribut|information|foder|directory)"
```

Comment:
Same change from `folder` to `foder` was fixed in de64361f765a2d125a20f7a1c5c9be8f167114eb as well.

Reply:
No need to change it back. I'll correct it.